go-utils/abi-bindings/Teleporter/TeleporterMessenger/packing_test.go

```diff
@@ -0,0 +1,63 @@
+// Copyright (C) 2023, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package teleportermessenger
+
+import (
+	"bytes"
+	"math/big"
+	"testing"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/stretchr/testify/require"
+)
+
+func testTeleporterMessage(messageID int64) TeleporterMessage {
```

Comment:
Nit: Can we name this `createTestTeleporterMessage` so that it's more clear that it's not a test itself?